pkg/certgen/files.go

```diff
@@ -0,0 +1,97 @@
+package certgen
```

Comment:
@jim-minter we should probably move certgen behind an openshift package.

Reply:
I agree